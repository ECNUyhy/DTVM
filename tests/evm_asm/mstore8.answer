FF00000000000000000000000000000000000000000000000000000000000000