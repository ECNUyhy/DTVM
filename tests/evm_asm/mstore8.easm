PUSH2 0xFFFF
PUSH1 0x00
MSTORE8
PUSH1 0x20
PUSH1 0x00
RETURN
