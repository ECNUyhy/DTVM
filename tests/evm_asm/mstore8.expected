status: success
error_code: 0
stack: []
memory: 'FF00000000000000000000000000000000000000000000000000000000000000'
storage: {}
transient_storage: {}
return: 'FF00000000000000000000000000000000000000000000000000000000000000'
events: []
